frontend/www/js/omegaup/arena/events_socket.test.ts

```diff
@@ -228,6 +228,11 @@ describe('EventsSocket', () => {
     jest.runOnlyPendingTimers();
     await server?.connected;
 
+    onRankingChanged.mockReturnValueOnce({
```

Comment:
```suggestion
    (<jest.Mock<typeof onRankingChanged>>onRankingChanged).mockReturnValueOnce({
```

con esto ya no se debería quejar el compilador.

Reply:
Si siguió quejándose. La buena noticia es que en https://stackoverflow.com/questions/51495473/typescript-and-jest-avoiding-type-errors-on-mocked-functions#answer-58229293 encontré que en `ts-jest` existe la función [mocked](https://kulshekhar.github.io/ts-jest/docs/guides/test-helpers/#mockedtitem-t-deep--false) que sirve para no tener que hacer cast de tipos